src/plugins/template/tests/functional/op_reference/experimental_detectron_detection_prior_grid.cpp

```diff
@@ -94,6 +88,22 @@ class ReferenceExperimentalPGGLayerTest : public testing::TestWithParam<Experime
         return result.str();
     }
 
+    virtual void Validate() override {
+        if (const auto comparison_size = GetParam().actualComparisonSize) {
+            ASSERT_EQ(executableNetwork.outputs().size(), refOutData.size());
+            actualOutData.clear();
+            for (const auto& output : executableNetwork.outputs())
+                actualOutData.emplace_back(inferRequest.get_tensor(output));
+
+            const auto shape = Shape{comparison_size};
+            const auto expected = Tensor{refOutData[0].get_element_type(), shape, refOutData[0].data()};
+            const auto inferred = Tensor{actualOutData[0].get_element_type(), shape, actualOutData[0].data()};
```

Comment:
```suggestion
            const auto& expected = refOutData[0];
            const auto& inferred = actualOutData[0];
```
?

Reply:
It won't work - the `shape` matters. The trick here is that hard-coded expected data is "shorter" than runtime inferred data. This is due to huge size of the tensor and this test provides a part of reference values to avoid (I presume) huge file size. Aim of this PR isn't to change the logic of the test.

Reply:
Added a comment to avoid confusion.